apps/s_client.c

```diff
@@ -2322,7 +2344,31 @@ int s_client_main(int argc, char **argv)
             BIO *fbio = BIO_new(BIO_f_buffer());
 
             BIO_push(fbio, sbio);
-            BIO_printf(fbio, "CONNECT %s HTTP/1.0\r\n\r\n", connectstr);
+            BIO_printf(fbio, "CONNECT %s HTTP/1.0\r\n", connectstr);
+	    /* 
```

Comment:
Style : indent of this block is not aligned with existing code.

Reply:
Thanks - my bad. A few TABs ended up in there, which throws out the indents.
Replaced all tabs with spaces.